action.yml

```diff
@@ -53,11 +53,14 @@ runs:
         fi
         if [ -f output_file ]; then
           echo "IS_CHANGE=true" >> $GITHUB_ENV
-          if [ -d snap ]; then
-            mv output_file snap/snapcraft.yaml
-          else
-            mv output_file snapcraft.yaml
-          fi
+          if [ -f rockcraft.yaml ]; then
+            mv output_file rockcraft.yaml
+        else
+            if [ -d snap ]; then
+                mv output_file snap/snapcraft.yaml
+            else
+                mv output_file snapcraft.yaml
+            fi
         fi
```

Comment:
If the previous 'else' was incorrectly indented, then a 'fi' is missing just between these two.